### ISSUE TITLE
Compress fonts [was; Compress WOFF and PNG fonts?]

### ISSUE THREAD

**Issue by TomasRiker** [mathjax/MathJax#1826]
I ran [woff-compress](https://github.com/hn/woff-compress) on the WOFF fonts and [zopflipng](https://github.com/google/zopfli) on the PNG fonts and achieved a size reduction of ~3.5% for WOFF files and ~25% for PNG files (I don't know if the PNG fonts are still relevant nowadays). 3.5% is not much, but it has to be done only once and thus might still be worth doing to save some bandwidth.

**Comment by pkra** [comment]
> ~3.5% for WOFF 

Nice. We are re-working the fonts for v3 and that will be useful to do.

Also, WOFF2 is important (see the relevant branch for a test).

> I don't know if the PNG fonts are still relevant nowadays

This came up in https://github.com/mathjax/MathJax/pull/1613. 

They are rarely used these days (badly served webfonts, user settings blocking webfonts). The review of a change would be quite time consuming (and necessary because bugs in zopfli compressors do exist). The PNG fonts will also be dropped from v3. 

Overall, it seems not worth spending the necessary resources.

**Comment by dpvc** [comment]
Agreed: the png fonts will not be part of v3, and are rarely used even in v2, so not worth spending the effort on them.

**Comment by pkra** [comment]
Renamed and re-labeled.

**Comment by sanmai-NL** [comment]
See #1990.

**Comment by dpvc** [comment]
Closed as part of a review of old issues.  MathJax v4.0 now uses WOFF2, and no longer has png versions.
